### PR TITLE
FEATURE: Automatically update MaxMindDb every 7 days.

### PR DESCRIPTION
**Pull request by nbianca**

### CODE REVIEW

File: app/jobs/scheduled/update_maxminddb.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+require_dependency 'discourse_ip_info'
+
+module Jobs
+  class UpdateMaxMindDb < Jobs::Scheduled
+    every 7.days
+
+    def execute(args)
+      DiscourseIpInfo.update!
+      DiscourseIpInfo.reload
```
Comment (ZogStriP) [inline app/jobs/scheduled/update_maxminddb.rb]: I think `update!` should take care of calling `reload`.

File: lib/discourse_ip_info.rb [hunk 2]
```diff
@@ -83,11 +90,29 @@ def get(ip, locale: :en, resolve_hostname: false)
...
-  def self.open_db(path)
-    instance.open_db(path)
-  end
-
   def self.get(ip, locale: :en, resolve_hostname: false)
     instance.get(ip, locale: locale, resolve_hostname: resolve_hostname)
   end
+
+  def self.download_mmdb(name)
+    uri = URI("http://geolite.maxmind.com/download/geoip/database/#{name}.tar.gz")
```
Comment (ZogStriP) [inline lib/discourse_ip_info.rb]: We should use https if it's available.

**Comment by SamSaffron** [comment]
I am so so sorry, but I am closing this, I don't feel we need to carry a job for this, if people want to update out of band they can run the rake task. We ship a new image every few months and I feel that is good enough cadence for regular updates.

**Comment by itsbhanusharma** [comment]
@SamSaffron I can't find the appropriate rake task to update maxminddb Please help?

**Pull request closed without merging**